apps/arweave/src/ar_data_sync.erl

```diff
@@ -459,12 +459,17 @@ get_chunk(Offset, #{ packing := Packing } = Options) ->
 			ModuleIDs = [ar_storage_module:id(Module) || Module <- Modules],
 			RootRecords = [ets:lookup(sync_records, {ar_data_sync, ID})
 					|| ID <- ModuleIDs],
-			log_chunk_error(RequestOrigin, chunk_record_not_found,
-					[{modules_covering_offset, ModuleIDs},
-					{root_sync_records, RootRecords},
-					{seek_offset, SeekOffset},
-					{reply, io_lib:format("~p", [Reply])},
-					{is_recorded_unpacked, io_lib:format("~p", [UnpackedReply])}]),
+			case RequestOrigin of
```

Comment:
`log_chunk_error` should already skip logging for `http` and `tx_data` origins - is there an origin that I missed? I can add it to the matching clause

https://github.com/ArweaveTeam/arweave/pull/722/files#diff-2934ddad2ed46ac194a6be7abde4be6d440bc7565e1128bba234fad9fd2bd5d2L1489-L1494